content/docs/introduction/glossary.md

```diff
@@ -51,6 +51,14 @@ metrics that are exposed in a non-Prometheus format into a format Prometheus sup
 
 An instance is a label that uniquely identifies a target in a job.
 
+### Label
+
+A label is a key-value pair you can associate to your metric, allowing to provide more details by appending those additional fields.
```

Comment:
Good idea for an addition, but this doesn't fit with the writing style.

How about this:
```
A label is a key-value pair associated with a metric. They allow for identification of different instances of the same metric.

For example, different CPUs on a node, or different nodes in a cluster.
```

Reply:
Sounds good!